components/frontend/src/report/ReportDashboard.js

```diff
@@ -78,14 +79,30 @@ export function ReportDashboard({ dates, measurements, onClick, onClickTag, relo
                     key={tag}
                     maxY={nrMetrics}
                     onClick={() => onClickTag(tag)}
+                    selected={anyTagsHidden}
                     summary={summary}
                 />
             )
         })
     }
+    const extraCards = []
+    if (report.issue_tracker?.type && !settings.hiddenCards.includes("issues")) {
+        const selected = settings.metricsToHide.value === "no_issues"
+        extraCards.push(
+            <IssuesCard
+                key="issues"
+                onClick={() => settings.metricsToHide.set(selected ? "none" : "no_issues")}
+                report={report}
+                selected={selected}
+            />,
+        )
+    }
+    if (!settings.hiddenCards.includes("legend")) {
```

Comment:
Ew... "if hidden does not include". But it is what it is I guess

Reply:
I'll add an `excludes()` to ` useArrayURLSearchQuery`, that's just a oneliner and should help with readability.